recipe/meta.yaml

```diff
@@ -32,7 +32,7 @@ requirements:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - patch  # [unix]
```

Comment:
The two patches above are without selectors. Should we add `m2-patch` for `win` too?

Reply:
`conda-build` itself causes `m2-patch` to be pulled in for `win-64`, but, yes I think its good to be explicit, thanks 👍

Reply:
Hm, not sure why yet but it doesn't build if I add that, so I reverted it.